src/main.tsx

```
@@ -1,10 +1,27 @@
-import './index.css'
-import App from './App.tsx'
+import { createRoot } from 'react-dom/client';
+import { BrowserRouter, Route, Routes } from 'react-router';
+import './index.css';
+import App from './App';
+import { DesignSystemModule, LandingPageModule } from './modules';
+import { PreviewTalentModule } from './modules/PreviewTalentModule';
+import { RegisterModule } from './modules/RegisterFormModule';
+import LoginModule from './modules/LoginFormModule';
+import { TalentProfilePage } from './modules/TalentProfileModule';
+import { TalentProfilePage } from './modules/TestCertifModule';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TalentProfilePage'. Consider to delete it or rename it.

'TalentProfilePage' is defined here:


(lint/suspicious/noRedeclare)